### ISSUE TITLE
Switch /team to /contributors ?

### ISSUE THREAD

**Issue by philwebb** [spring-io/sagan#610]
Inspired by [this tweet](https://twitter.com/mojavelinux/status/671916562446553088) I wonder if we shouldn't change `/team` to `/contributors` (or add an additional page) so that we can highlight people that contribute, even if they don't work for us?

**Comment by cbeams** [comment]
One idea when /team was originally being designed was to pull contributor information from the GitHub API. Imagine a union of all contributors across all official Spring projects, ranked by total number of commits. Then determine which among them are actually employees, by seeing which are members of the spring-projects organization (or some more specific GitHub team). Give the employees some kind of promoted status (including showing them on the map).

As far as renaming /team to /contributors, I'm not sure it's ideal. "Spring Team" has a certain value that would be lost if the page were renamed to "Spring Contributors". The overall team page could have what it has now (active team members, i.e. employees), but could be enhanced with an alumni section (which has been talked about before) and below that a contributors section as described above. Not exactly a perfect way to organize this information, but perhaps worth keeping it all in one place while still focusing primarily on "the team".

btw, the tweet link above is 404.

**Comment by philwebb** [comment]
Huh, tweet has been deleted for some reason. It basically said that a `/contributors` page is sensible thing for any project to have. Here's the asciidoctor one: http://asciidoctor.org/contributors/

**Comment by cbeams** [comment]
Yeah, could just have two pages in the end: /team (perhaps including 'alumni' at bottom) and a new page, /contributors.

**Comment by philwebb** [comment]
I think two pages actually might make the most sense. They really have quite different purposes in the end (for example team includes twitter handles, blog posts etc). The team page could be updated to include something to highlight how important contributors are provide a link.

The contributors page could be auto-generated from commit stats and perhaps filtered to remove active team members. Limiting by project would also be nice option.
